spec/integration/hanami/controller/full_stack_spec.rb

```diff
@@ -14,13 +14,6 @@ def app
     expect(last_response.body).to include(':greeting=>"Hello"')
   end
 
-  it "omits the body if the request is HEAD" do
```

Comment:
@GustavoCaso Instead of removing this spec, I'd keep it, but update with the new behavior.